src/plugins/cachefilter/README.md

```diff
@@ -0,0 +1,16 @@
+- infos = Information about the cachefilter plugin is in keys below
+- infos/author = Marvin Mall <e1225943@student.tuwien.ac.at>
```

Comment:
Yes of course, but the student e-mail address will vanish after you finished. What about a `@libelektra.org` address where you can choose to which address the e-mails go to? Which address do you want?